test/validation/api/classification/odp_classification_common.c

```diff
@@ -389,6 +417,22 @@ odp_packet_t create_packet(cls_packet_info_t pkt_info)
 			ODPH_ERR("odph_udp_tcp_chksum failed\n");
 			return ODP_PACKET_INVALID;
 		}
+	} else if (pkt_info.gtpu) {
+		udp->src_port = odp_cpu_to_be_16(ODP_GTPU_UDP_PORT);
```

Comment:
I think it would be better to have a different source port (what if there is a bug that the classifier looks at the source port instead of the destination port), at least in the test that should result a match.

Reply:
Ack